attestations-collections/local/hydra-s1-local.ts

```diff
@@ -34,6 +34,13 @@ export const hydraS1LocalAttester = generateHydraS1Attester(
           await groupStore.latest("local-group"),
         ],
       },
+      // top 100 ens users
```

Comment:
You need to remove the modification of the hydra-s1-local in order to have only a diff for playground